### PR TITLE
[MB-6424] Display readable shipment type in Move Task Order and Payment Requests

### PR DESCRIPTION
**Pull request by noelledusahel**
## Description

As an office user, I should see the shipment types displayed as a readable display of the enum value
Places impacted: 
- Move Details
- Move Details - Shipment approval popup
- MTO page
- Payment Request Details
- Payment service items review cards

- [ ] HHG, HHG_LONGHAUL_DOMESTIC, HHG_SHORTHAUL_DOMESTIC display as Household Goods
- [ ] NTS, HHG_INTO_NTS_DOMESTIC display as Non-temp storage
- [ ] NTSR, HHG_OUTOF_NTS_DOMESTIC display as Non-temp storage release 
- [ ] Correct the Move details display for NTSR, change to NTS-R

## Reviewer Notes
changes made on the following pages/modals
### Move Details
Screen Shot 2021-01-27 at 10 32 08 AM
### Move details shipment approval modal
Screen Shot 2021-01-26 at 1 22 28 PM
### Move Task Order
Screen Shot 2021-01-26 at 1 22 50 PM

### Payment Request Details
Screen Shot 2021-01-27 at 10 45 04 AM
### Payment Service Items Review
Screen Shot 2021-01-26 at 3 59 36 PM
 Review 

## Setup

Add any steps or code to run in this section to help others prepare to run your code:

```sh
make db_dev_e2e_populate
make server_run
make office_client_run
```

## Code Review Verification Steps

* [ ] If the change is risky, it has been tested in experimental before merging.
* [ ] Code follows the guidelines for [Logging](https://github.com/transcom/mymove/wiki/Backend-Programming-Guide#logging)
* [ ] The requirements listed in
 [Querying the Database Safely](https://github.com/transcom/mymove/wiki/Backend-Programming-Guide#querying-the-database-safely)
 have been satisfied.
* Any new migrations/schema changes:
  * [ ] Follow our guidelines for zero-downtime deploys (see [Zero-Downtime Deploys](https://github.com/transcom/mymove/wiki/migrate-the-database#zero-downtime-migrations))
  * [ ] Have been communicated to #g-database
  * [ ] Secure migrations have been tested following the instructions in our [docs](https://github.com/transcom/mymove/wiki/migrate-the-database#secure-migrations)
* [ ] There are no aXe warnings for UI.
* [ ] This works in [Supported Browsers and their phone views](https://github.com/transcom/mymove/tree/master/docs/adr/0016-Browser-Support.md) (Chrome, Firefox, IE, Edge).
* [ ] Tested in the Experimental environment (for changes to containers, app startup, or connection to data stores)
* [ ] User facing changes have been reviewed by design.
* [ ] Request review from a member of a different team.
* [ ] Have the Jira acceptance criteria been met for this change?

## References

* [Jira story](https://dp3.atlassian.net/secure/RapidBoard.jspa?rapidView=27&modal=detail&selectedIssue=MB-6424) for this change
* [this article](tbd) explains more about the approach used.

## Screenshots

If this PR makes visible UI changes, an image of the finished UI can help reviewers and casual
observers understand the context of the changes. A before image is optional and
can be included at the submitter's discretion.

Consider using an animated image to show an entire workflow instead of using multiple images. You may want to use GIPHY CAPTURE for this! 📸

_Please frame screenshots to show enough useful context but also highlight the affected regions._

### CODE REVIEW

File: src/components/Office/ShipmentApprovalPreview.jsx [hunk 1]
```diff
@@ -4,7 +4,7 @@ import * as PropTypes from 'prop-types';
 import classNames from 'classnames';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 
-import { mtoShipmentTypeToFriendlyDisplay } from '../../shared/formatters';
+// import { mtoShipmentTypeToFriendlyDisplay } from '../../shared/formatters';
```
Comment (duncan-truss) [inline src/components/Office/ShipmentApprovalPreview.jsx:7]: looks like you can delete this now

Comment (noelledusahel) [reply src/components/Office/ShipmentApprovalPreview.jsx:7]: 🤦‍♀️

File: src/components/Office/ShipmentApprovalPreview.test.jsx [hunk 2]
```diff
@@ -219,6 +219,9 @@ describe('Shipment preview modal', () => {
     expect(wrapper.find(ShipmentContainer).exists()).toBe(true);
     expect(wrapper.find(AllowancesTable).exists()).toBe(true);
     expect(wrapper.find(CustomerInfoTable).exists()).toBe(true);
+
+    expect(wrapper.text().includes('Household goods')).toBeTruthy();
+    expect(wrapper.text().includes('Non-temp storage release')).toBeTruthy();
```
Comment (duncan-truss) [inline src/components/Office/ShipmentApprovalPreview.test.jsx:224]: Could we use a find selector here so we're not searching the whole dom for a text string?

File: src/pages/Office/MoveTaskOrder/MoveTaskOrder.test.jsx [hunk 3]
```diff
@@ -52,10 +52,29 @@ jest.mock('hooks/queries', () => ({
...
+          scheduledPickupDate: '2020-03-16',
+          requestedPickupDate: '2020-03-15',
+          pickupAddress: {
+            street_address_1: '932 Baltic Avenue',
+            city: 'Chicago',
+            state: 'IL',
+            postal_code: '60601',
+          },
+          destinationAddress: {
+            street_address_1: '10 Park Place',
```
Comment (duncan-truss) [inline src/pages/Office/MoveTaskOrder/MoveTaskOrder.test.jsx:67]: 🎩 🐕 🏨 🚋 ⚡

File: src/pages/Office/MoveTaskOrder/MoveTaskOrder.test.jsx [hunk 4]
```diff
@@ -90,6 +109,8 @@ describe('MoveTaskOrder', () => {
 
   it('should render the ShipmentHeading', () => {
     expect(wrapper.find('ShipmentHeading').exists()).toBe(true);
+    expect(wrapper.text().includes('Household goods')).toBeTruthy();
+    expect(wrapper.text().includes('Non-temp storage')).toBeTruthy();
```
Comment (duncan-truss) [inline src/pages/Office/MoveTaskOrder/MoveTaskOrder.test.jsx:113]: Is this test supposed to be for one shipment or multiple?

**Review by jenniferlynparsons — COMMENTED** [review]
Looks good other than test cleanup suggestions.

File: src/pages/Office/MoveTaskOrder/MoveTaskOrder.test.jsx (hunk 4 above)
Comment (noelledusahel) [reply src/pages/Office/MoveTaskOrder/MoveTaskOrder.test.jsx:113]: i added a NTS shipment, so its meant for multiple

**Review by brooksr4 — APPROVED** [review]
Labels seems to be matching correctly

**Review by jenniferlynparsons — APPROVED** [review]
LGTM!

**Review by duncan-truss — APPROVED** [review]

**Pull request merged**